cfme/tests/automate/test_domain.py

```diff
@@ -187,7 +178,7 @@ def test_domain_name_wrong(appliance):
 
 
 @pytest.mark.tier(2)
-def test_domain_lock_unlock(request, appliance):
+def test_domain_lock_unlock(domain, appliance):
```

Comment:
Optional: but fiy @ganeshhubale when we have have some object entity and for small use-case `appliance` you can get with `entity(domain).appliance`. Not required for this implementation.